examples/rust/file_transfer/tests/100_32_test.bin

```diff
@@ -0,0 +1,2 @@
+�_��	��Vr����P���K�m���4�k�H�y\r\��g�c��"�_��0���}Lo����zi���ڠ;"N��Y�E��$ɥ���|
```

Comment:
Looks like elfish

Reply:
Grrr, those files snuck in because I was running the tests in parallel....